### PR TITLE
Run on ack

### PR DESCRIPTION
**Pull request by hxdtest**
provide an train case to run on ack

### CODE REVIEW

File: dlrover/trainer/constants/tf_constants.py [hunk 1]
```diff
@@ -62,3 +62,5 @@ class TFConstants(object):
     EstimatorTrainingHooks = Constant("training_hooks")
     EstimatorPredictionHooks = Constant("prediction_hooks")
     EstimatorEvaluationHooks = Constant("evaluation_hooks")
+    EnableEasydl = Constant("enable_easydl")
```
Comment (workingloong) [inline dlrover/trainer/constants/tf_constants.py:65]: It is better to rename `enable_easydl` to `enable_auto_scaling`.  The name "easydl" will be abandoned in DLRover.

File: dlrover/trainer/examples/estimator_executor/elasticjobtf2.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: main
+              image: easydl/tf-estimator:iris_dnn_v0
+              imagePullPolicy: IfNotPresent
+              command:
+                - /bin/bash
+                - -c
+                - "pip uninstall dlrover -y && pip install -i https://test.pypi.org/simple/ dlrover==0.1.0rc0.dev6 -U && pip install pyhocon  && cd /usr/local/lib/python3.8/site-packages/dlrover/trainer/examples/estimator_executor  && python -m dlrover.trainer.entry.local_entry --platform=Kubernetes  --conf=conf.TrainConf --enable_easydl=True --ps_num=1 --worker_num=2 --evaluator_num=1" # yamllint disable-line rule:line-length
```
Comment (workingloong) [inline dlrover/trainer/examples/estimator_executor/elasticjobtf2.yaml:25]: we can use  `pip install -i https://test.pypi.org/simple/ dlrover==0.1.0rc0.dev6 -I` to replace 
`pip uninstall dlrover -y && pip install -i https://test.pypi.org/simple/ dlrover==0.1.0rc0.dev6 -U`

Comment (workingloong) [reply dlrover/trainer/examples/estimator_executor/elasticjobtf2.yaml:25]: Can we not set `--ps_num=1 --worker_num=2 --evaluator_num=1` if using auto-scaling?

Comment (hxdtest) [reply dlrover/trainer/examples/estimator_executor/elasticjobtf2.yaml]: if using auto-scaling, `--ps_num=1 --worker_num=2 --evaluator_num=1` can be removed.

File: dlrover/trainer/constants/tf_constants.py (hunk 1 above)
Comment (hxdtest) [reply dlrover/trainer/constants/tf_constants.py]: enable_auto_scaling is added

**Review by workingloong — APPROVED** [review]
LGTM

**Pull request merged**